crates/kornia-pnp/Cargo.toml

```diff
@@ -21,6 +21,14 @@ glam = { version = "0.30.0", default-features = false }
 thiserror = { workspace = true }
 
 nalgebra = "0.32"
+rand = "0.8"
```

Comment:
add this to the workspace level as it's very generic one. Can you group and keep alphabetical order of the deps ?